app/src/main/java/at/gammastrahlung/monopoly_app/network/MonopolyClient.java

```diff
@@ -157,22 +137,81 @@ public void cheating(int value1, int value2) {
         gameData.getDice().setValue1(value1);
         gameData.getDice().setValue2(value2);
 
-        webSocketClient.sendMessage(ClientMessage.builder()
-                .messagePath("cheating")
-                .message(String.valueOf(value1 + value2))
-                .player(gameData.getPlayer())
-                .build());
+        webSocketClient.sendMessage(ClientMessage.builder().messagePath("cheating").message(String.valueOf(value1 + value2)).player(gameData.getPlayer()).build());
     }
 
     /**
      * Sends a game_state message to the server.
+     *
      * @param gameId The ID of the game
      */
     public void getGameState(int gameId) {
+        webSocketClient.sendMessage(ClientMessage.builder().messagePath("game_state").message(String.valueOf(gameId)).player(GameData.getGameData().getPlayer()).build());
```

Comment:
Lombok builder is unreadable when written in a single line please reformat (IntelliJ automatically can do this).
Also do this for further `ClientMessage.builder()` calls